tap.go

```diff
@@ -209,7 +221,16 @@ func DecodeValueWithXattrs(data []byte) ([]byte, []Xattr, error) {
 		if len(components) != 3 {
 			return nil, nil, fmt.Errorf("Unexpected number of components found in xattr pair: %s", pairBytes)
 		}
-		xattrs = append(xattrs, Xattr{string(components[0]), components[1]})
+		xattrKey := string(components[0])
+		for _, xattrName := range xattrNames {
+			if xattrName == xattrKey {
+				xattrs[xattrName] = components[1]
```

Comment:
Looks like we could `break` here once we've found a match for the name.

Reply:
Thanks